File: app/src/main/java/co/shek/kkiapayandroidsample/MyApp.java
package co.shek.kkiapayandroidsample;

import android.app.Application;
import co.opensi.kkiapay.uikit.Kkiapay;
import co.opensi.kkiapay.uikit.SdkConfig;

public class MyApp extends Application {
    @Override
    public void onCreate () {
        super.onCreate ();
        //Initialisation de l'API
        Kkiapay.init(this,
                "<PUBLIC-API-KEY>",
                new SdkConfig (R.raw.armoiries, R.color.colorPrimary, false));
    }
}

File: app/src/main/java/co/shek/kkiapayandroidsample/kotlin/MyApp.kt
package co.shek.kkiapayandroidsample.kotlin

import android.app.Application
import co.opensi.kkiapay.uikit.Kkiapay
import co.opensi.kkiapay.uikit.SdkConfig
import co.shek.kkiapayandroidsample.R

class MyApp : Application() {
    override fun onCreate() {
        super.onCreate()
        //Initialisation de l'API
        Kkiapay.init(applicationContext,
                "<PUBLIC-API-KEY>",
            SdkConfig(themeColor = R.color.colorPrimary,
                imageResource = R.raw.armoiries,
                    /* set enableSandbox = false in case you are using live API Keys */
                    enableSandbox = false)
        )
    }
}
